### PR TITLE
Adds pet pens to Head of Staff offices

### LINKED ISSUE #3664: HoS lacks a pet
#### Brief description of the issue
HoS doesn't have a pet for their office

#### What you expected to happen
To be on par with other command staff

#### What actually happened
I have to use Junior Officers as pets, and that results in ethics violations

#### Steps to reproduce
- Be HoS
- Cry 

#### Code Revision
- 849c1374a1b064b9c9a1fb8c1c43ed19a4259100
 
#### Anything else you may wish to add:
- I vote a spider

### PR DESCRIPTION
**Pull request by Mewchild**
- Makes the CMO office larger and comfier, adds pen for Runtime
- Adds a pen for Fluffy behind the QM office, slightly reworks maint to accommodate. Adds a second Diyaab (Silky) to be lore friendly
- Makes the Head of Staff meeting room slightly larger (to center the table) on request of SweetBlueSylveon
- Adds a pen to the CE office for Noodle
- Adds a pen to the RD office for Kendrick
- Adds a Pen to the HoS office for their new pet, Scruffy. Scruffy is a Shantak (a Polaris Expeditionary mob) with it's AI modified to be passive. Description is: "A piglike creature with a bright iridiscent mane that sparkles as though lit by an inner light. Don't be fooled by its beauty though."

image

HoP and CD offices are unchanged. I'll be making a future PR once I figure out how I want to do them, as I didn't see any elegant solutions

Closes #3664

### CODE REVIEW

**Comment by Mewchild** [comment]
Behave, Travis

**Comment by SpadesNeil** [comment]
It's a nice idea but some of these feel out of place. A 'pen' to me is more like the glass window enclosure you have for only one of these station pets, while the rest are basically shoved in a closet.

The additions also appear mostly forced and tacked on in an unnatural way. The rooms themselves have no life to them and feel more like prison cells than pet pens.

**Comment by Mewchild** [comment]
I will say that for the HoS pet, the prison cell feel was absolutely the intention for it. I'll adjust the rest, however

**Comment by Wickedtemp** [comment]
So we won't have max-dirt tiles in all Head offices because of moving pets?

**Comment by Mewchild** [comment]
Pets leaving dirt behind was removed ages ago

**Comment by chaoko99** [comment]
And yet it's still an issue.

**Comment by Mewchild** [comment]
Should have been fixed in https://github.com/VOREStation/VOREStation/pull/4875

File: code/game/machinery/doors/brigdoors.dm [hunk 1]
```diff
@@ -351,6 +351,11 @@
 	name = "Cell 6"
 	id = "Cell 6"
 
+/obj/machinery/door_timer/tactical_pet_storags
```
Comment (TheFurryFeline) [inline code/game/machinery/doors/brigdoors.dm]: Just noticed this thing's a .dm file, as in Polaris. Might want to do a _vr addition?

**Pull request merged**